on_chain/test/user.ts

```diff
@@ -30,43 +30,46 @@ function loadEVMLinkData(jsonPath: string): EVMLinkData {
         const taBytes: Uint8Array = ethers.decodeBase64(j)
         const ta: string = Buffer.from(taBytes).toString('hex');
 
-        return {publicKey: `0x${publicKeyHex}`, transitiveAttestation: `0x${ta}`};
+        return {
+            publicKey: `0x${publicKeyHex}`,
+            transitiveAttestation: `0x${ta}`
+        };
     } catch (e) {
         throw new Error(`Error loading EVM link data: ` + e);
     }
 }
 
 interface UserContract extends ethers.Contract {
     // @ts-ignore
-    processTransitivelyAttestedHelloWorldOutput(publicKey: string, ta: string): Promise<ethers.ContractTransactionResponse>;
+    setEnclaveAttestedAppPubKey(publicKey:string): Promise<ethers.ContractTransactionResponse>;
+    processTransitiveAttestedFunctionCall(ta: string): Promise<ethers.ContractTransactionResponse>;
 }
 
-describe("Local Test", function (): void {
+describe("User contract test", function (): void {
```

Comment:
We only have one test now (no Sepolia), so I renamed it from "Local Test", which also gives a more informative message when running the test suite.